src/main/java/com/google/ehub/servlets/ItemSubmissionServlet.java

```diff
@@ -62,11 +69,30 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
                                                                     .setDirectors(directors)
                                                                     .setWriters(writers)
                                                                     .setActors(actors)
+                                                                    .setOmdbId(omdbId)
                                                                     .build());
 
     response.sendRedirect("/index.html");
   }
 
+  /**
+   * The response of the GET request contains a boolean value that tells whether the item is unique
+   * or not.
+   */
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String omdbId = request.getParameter(OMDB_ID_PARAMETER_KEY);
+
+    if (!isParameterValid(omdbId, MAX_CHARS)) {
+      System.err.println(
```

Comment:
You could also do : response.sendError(HttpServletResponse.SC_BAD_GATEWAY, "ItemSubmissionServlet: omdbId parameter given in Get Request is invalid!");

Reply:
+1 sending the error will inform the client that an error occurred